coa-helm/templates/cbas-ui.yaml

```diff
@@ -27,8 +27,6 @@ data:
     {
       "leoUrlRoot" : "{{ .Values.leonardo.url }}",
       "cbasUrlRoot" : "{{ .Values.relay.path }}/cbas",
-      "cromwellUrlRoot" : "{{ .Values.relay.path }}/cromwell",
-      "wdsUrlRoot" : "{{ .Values.relay.path }}/wds",
-      "wdsInstanceId" : "{{ .Values.persistence.workspaceManager.workspaceId }}"
+      "cromwellUrlRoot" : "{{ .Values.relay.path }}/cromwell"
```

Comment:
I think cbas will still need these wds values in order to communicate with it.

Reply:
I wasnt sure if the .Values.relay.path is still the right url root? after digging around I guess the url hasnt changes, just something else is deploying it, but in this context I guess I am not quite sure if it is safe to assume that Values.relay and Values.ingess are going to have the right path for wds now that the charts are decoupled?